tribute.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <title>Document1</title>
 </head>
 <body>
     <div class="container">
```

Comment:
Please also permit me to mention that, your document is good but follows no semantics. It's always recommended to use semantics tags in your HTML. You can use semantics tags like <header> to group related elements like the <nav> or a logo that appear on the head of your document. As a recommendation, please use semantic tags.